src/visible.js

```diff
@@ -102,10 +102,6 @@ eg.module("visible", ["jQuery", eg, document], function($, ns, doc) {
 				delay = -1;
 			}
 
-			if (typeof delay === "undefined") {
-				delay = -1;
-			}
-
```

Comment:
Thanks your contribution.
the existing code is not clear, so the modified code is also not clear.
@cimfalab Could you improve this code more clear?